browser/src/control/Control.Toolbar.js

```diff
@@ -318,163 +314,163 @@ function insertTable(grid = document.getElementsByClassName('inserttable-grid')[
 var shapes = {
 	'insertshapes': {
 		'Basic Shapes': [
-			{img: 'basicshapes_rectangle', uno: 'BasicShapes.rectangle'},
-			{img: 'basicshapes_round-rectangle', uno: 'BasicShapes.round-rectangle'},
-			{img: 'basicshapes_quadrat', uno: 'BasicShapes.quadrat'},
-			{img: 'basicshapes_round-quadrat', uno: 'BasicShapes.round-quadrat'},
-			{img: 'basicshapes_circle', uno: 'BasicShapes.circle'},
-			{img: 'basicshapes_ellipse', uno: 'BasicShapes.ellipse'},
-
-			{img: 'basicshapes_circle-pie', uno: 'BasicShapes.circle-pie'},
-			{img: 'basicshapes_isosceles-triangle', uno: 'BasicShapes.isosceles-triangle'},
-			{img: 'basicshapes_right-triangle', uno: 'BasicShapes.right-triangle'},
-			{img: 'basicshapes_trapezoid', uno: 'BasicShapes.trapezoid'},
-			{img: 'basicshapes_diamond', uno: 'BasicShapes.diamond'},
-			{img: 'basicshapes_parallelogram', uno: 'BasicShapes.parallelogram'},
-
-			{img: 'basicshapes_pentagon', uno: 'BasicShapes.pentagon'},
-			{img: 'basicshapes_hexagon', uno: 'BasicShapes.hexagon'},
-			{img: 'basicshapes_octagon', uno: 'BasicShapes.octagon'},
-			{img: 'basicshapes_cross', uno: 'BasicShapes.cross'},
-			{img: 'basicshapes_ring', uno: 'BasicShapes.ring'},
-			{img: 'basicshapes_block-arc', uno: 'BasicShapes.block-arc'},
-
-			{img: 'basicshapes_can', uno: 'BasicShapes.can'},
-			{img: 'basicshapes_cube', uno: 'BasicShapes.cube'},
-			{img: 'basicshapes_paper', uno: 'BasicShapes.paper'},
-			{img: 'basicshapes_frame', uno: 'BasicShapes.frame'}
+			{img: 'basicshapes_rectangle', uno: 'BasicShapes.rectangle', text: _('Rectangle')},
+			{img: 'basicshapes_round-rectangle', uno: 'BasicShapes.round-rectangle', text: _('Rectangle, Rounded')},
```

Comment:
can't these strings be fetched from the core?
please see `_UNO('...uno command')`